tests/io/aims/conftest.py

```diff
@@ -1,19 +1,164 @@
 from __future__ import annotations
 
-import os
+import gzip
+import json
+from glob import glob
+from pathlib import Path
+from typing import Any
 
+import numpy as np
 import pytest
+from monty.io import zopen
 
-from pymatgen.core import SETTINGS
-
-MODULE_DIR = os.path.dirname(__file__)
+from pymatgen.core import SETTINGS, Molecule, Structure
+from pymatgen.util.testing import TEST_FILES_DIR
 
 
 @pytest.fixture(autouse=True)
 def _set_aims_species_dir_env_var(monkeypatch: pytest.MonkeyPatch) -> None:
-    monkeypatch.setenv("AIMS_SPECIES_DIR", f"{MODULE_DIR}/species_directory")
+    monkeypatch.setenv("AIMS_SPECIES_DIR", f"{TEST_FILES_DIR}/io/aims/species_directory")
 
 
 @pytest.fixture(autouse=True)
 def _set_aims_species_dir_settings(monkeypatch: pytest.MonkeyPatch):
-    monkeypatch.setitem(SETTINGS, "AIMS_SPECIES_DIR", f"{MODULE_DIR}/species_directory")
+    monkeypatch.setitem(SETTINGS, "AIMS_SPECIES_DIR", f"{TEST_FILES_DIR}/io/aims/species_directory")
+
+
+def check_band(test_line: str, ref_line: str) -> bool:
```

Comment:
Need confirm: these helper functions for io.aims test is now relocated from `pymatgen.util.testing.aims`, is this intended? @shyuep